### DISCUSSION TITLE
posts/%E6%B3%B0%E5%8B%92%E5%85%AC%E5%BC%8F%E6%80%BB%E7%BB%93/

### DISCUSSION THREAD

**Discussion by giscus** [fffzlfk/fffzlfk.github.io#2] [General]
# posts/%E6%B3%B0%E5%8B%92%E5%85%AC%E5%BC%8F%E6%80%BB%E7%BB%93/

Taylor formula

http://localhost:1313/posts/%E6%B3%B0%E5%8B%92%E5%85%AC%E5%BC%8F%E6%80%BB%E7%BB%93/

**Comment by fffzlfk** [comment]
test
